Authentication/Login/View/LoginController.swift

```diff
@@ -71,35 +68,35 @@ public final class LoginController: UIViewController {
         view.addGestureRecognizer(tapRecognizer)
     }
     
-    public override func viewWillAppear(animated: Bool) {
+    public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.navigationBarHidden = true
+        navigationController?.isNavigationBarHidden = true
     }
     
 }
 
 private extension LoginController {
     
-    private func bindViewModel() {
+    func bindViewModel() {
```

Comment:
I can't, because `LoginController` overrides `viewDidLoad()` and calls `bindViewModel()`.